README.md

```diff
@@ -41,47 +46,79 @@ See the [changelog](CHANGELOG.md) for the latest release. You might also find us
     * `xml.java.home` in VSCode preferences
     * `java.home` in VSCode preferences
     * Environment variable `JAVA_HOME` or `JDK_HOME`
-    * **Note**: The path should end at the parent folder that contains the `bin` folder.
+    * **Note**: The path should end at the parent folder that contains the `bin` folder.  
       **Example Path**: `/usr/lib/jvm/java-1.8.0` if `bin` exists at `/usr/lib/jvm/java-1.8.0/bin`.
     * **Note**: If the path is not set, the extension will attempt to find the path to the JDK or JRE.
 
+See [how to set java home](https://github.com/redhat-developer/vscode-xml/blob/master/docs/Preferences.md#java-home) for more information how this extension searches for Java.
```

Comment:
The sentence is fine, but I think we don't need the bullet points above - details are found in the linked entry and directly below in Supported VS Code settings `xml.java.home`.

Reply:
Even if I understand that there are some duplication, I would kike to keep it because this section is about ALL settings. If user starts reading this section, he will not see that it exists xml.java.home.